ERROR - 2025-05-01 17:36:02 --> mysqli_sql_exception: Duplicate entry 'ART001' for key 'ref' in C:\xampp1\htdocs\AuTestWin\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\AuTestWin\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `ar...', 0)
#1 C:\xampp1\htdocs\AuTestWin\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ar...')
#2 C:\xampp1\htdocs\AuTestWin\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ar...')
#3 C:\xampp1\htdocs\AuTestWin\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#4 C:\xampp1\htdocs\AuTestWin\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp1\htdocs\AuTestWin\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp1\htdocs\AuTestWin\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp1\htdocs\AuTestWin\tests\ArticleModelTest.php(27): CodeIgniter\Model->insert(Array)
#8 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(1656): App\Tests\Models\ArticleModelTest->testInsertArticle()
#9 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(514): PHPUnit\Framework\TestCase->runTest()
#10 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestRunner\TestRunner.php(87): PHPUnit\Framework\TestCase->runBare()
#11 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(361): PHPUnit\Framework\TestRunner->run(Object(App\Tests\Models\ArticleModelTest))
#12 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestSuite.php(369): PHPUnit\Framework\TestCase->run()
#13 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(64): PHPUnit\Framework\TestSuite->run()
#14 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\TextUI\Application.php(210): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\TextUI\Configuration\Configuration), Object(PHPUnit\Runner\ResultCache\DefaultResultCache), Object(PHPUnit\Framework\TestSuite))
#15 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\phpunit(104): PHPUnit\TextUI\Application->run(Array)
#16 C:\xampp1\htdocs\AuTestWin\vendor\bin\phpunit(122): include('C:\\xampp1\\htdoc...')
#17 {main}
ERROR - 2025-05-01 17:36:02 --> mysqli_sql_exception: Duplicate entry 'ART002' for key 'ref' in C:\xampp1\htdocs\AuTestWin\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\AuTestWin\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `ar...', 0)
#1 C:\xampp1\htdocs\AuTestWin\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ar...')
#2 C:\xampp1\htdocs\AuTestWin\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ar...')
#3 C:\xampp1\htdocs\AuTestWin\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#4 C:\xampp1\htdocs\AuTestWin\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp1\htdocs\AuTestWin\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp1\htdocs\AuTestWin\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp1\htdocs\AuTestWin\tests\ArticleModelTest.php(40): CodeIgniter\Model->insert(Array)
#8 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(1656): App\Tests\Models\ArticleModelTest->testUniqueReference()
#9 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(514): PHPUnit\Framework\TestCase->runTest()
#10 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestRunner\TestRunner.php(87): PHPUnit\Framework\TestCase->runBare()
#11 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(361): PHPUnit\Framework\TestRunner->run(Object(App\Tests\Models\ArticleModelTest))
#12 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestSuite.php(369): PHPUnit\Framework\TestCase->run()
#13 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(64): PHPUnit\Framework\TestSuite->run()
#14 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\TextUI\Application.php(210): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\TextUI\Configuration\Configuration), Object(PHPUnit\Runner\ResultCache\DefaultResultCache), Object(PHPUnit\Framework\TestSuite))
#15 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\phpunit(104): PHPUnit\TextUI\Application->run(Array)
#16 C:\xampp1\htdocs\AuTestWin\vendor\bin\phpunit(122): include('C:\\xampp1\\htdoc...')
#17 {main}
ERROR - 2025-05-01 18:06:14 --> mysqli_sql_exception: Duplicate entry 'ART002' for key 'ref' in C:\xampp1\htdocs\AuTestWin\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp1\htdocs\AuTestWin\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `ar...', 0)
#1 C:\xampp1\htdocs\AuTestWin\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ar...')
#2 C:\xampp1\htdocs\AuTestWin\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ar...')
#3 C:\xampp1\htdocs\AuTestWin\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ar...', Array, false)
#4 C:\xampp1\htdocs\AuTestWin\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp1\htdocs\AuTestWin\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp1\htdocs\AuTestWin\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp1\htdocs\AuTestWin\tests\ArticleModelTest.php(50): CodeIgniter\Model->insert(Array)
#8 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(1656): App\Tests\Models\ArticleModelTest->testUniqueReference()
#9 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(514): PHPUnit\Framework\TestCase->runTest()
#10 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestRunner\TestRunner.php(87): PHPUnit\Framework\TestCase->runBare()
#11 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestCase.php(361): PHPUnit\Framework\TestRunner->run(Object(App\Tests\Models\ArticleModelTest))
#12 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\Framework\TestSuite.php(369): PHPUnit\Framework\TestCase->run()
#13 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(64): PHPUnit\Framework\TestSuite->run()
#14 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\src\TextUI\Application.php(210): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\TextUI\Configuration\Configuration), Object(PHPUnit\Runner\ResultCache\DefaultResultCache), Object(PHPUnit\Framework\TestSuite))
#15 C:\xampp1\htdocs\AuTestWin\vendor\phpunit\phpunit\phpunit(104): PHPUnit\TextUI\Application->run(Array)
#16 C:\xampp1\htdocs\AuTestWin\vendor\bin\phpunit(122): include('C:\\xampp1\\htdoc...')
#17 {main}
